src/everest/simulator/everest_to_ert.py

```diff
@@ -336,10 +336,13 @@ def _is_dir_all_geo(source, ever_config: EverestConfig):
                 "Expected source to exist for data installation, "
                 "did not find: {}".format(geo_source)
             )
-            raise AssertionError(msg)
+            raise ValueError(msg)
 
         is_dir.append(os.path.isdir(geo_source))
 
+    if not is_dir:
```

Comment:
Is this necessary? Or am I missing something (or is it a better safe than sorry)?

Reply:
This is fixing a small bug I noticed while testing out the functionality. But I guess it should not be part of this PR I will add it in a separate PR.